CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Keycloak Community
+
+Keycloak is an Open Source Identity and Access Management solution for modern Applications and Services. Keycloak Benchmark is the benchmarking tool supported by the Keycloak Community.
+
+## Building and working with the codebase
+
+Question: Not sure if we should provide more details about this section or have the users pointed to our documentation
```

Comment:
@ahus1 I did have a question on this aspect., on how to add details about building and working with the code base. We don't have an explicit section for it in the docs., but individual pieces needs different build mechanisms. Not sure how to approach this.

Reply:
appreciate it.